R/parser.R

```diff
@@ -4,6 +4,136 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at https://mozilla.org/MPL/2.0/.
 
+#' Parse Bugzilla issues data obtained from json files from Bugzilla crawler
```

Comment:
Use \code{\link{}} to explicitly refer to Leilani function and so when the user uses your function docs they can click and jump to it. 

In addition, use @seealso and include Leilani function there too. I have mentioned this in other revisions and I believe e-mails.